app/views/events/_form.html.erb

```diff
@@ -22,28 +22,13 @@
   </div>
 
   <div class="field">
-    <%= form.label :gamemode %>
-    <%= form.text_field :gamemode, id: :event_gamemode %>
+    <%= form.label :game_mode %>
+    <%= form.text_field :game_mode, id: :event_game_mode %>
   </div>
 
   <div class="field">
-    <%= form.label :sport %>
-    <%= form.text_field :sport, id: :event_sport %>
-  </div>
-
-  <div class="field">
-    <%= form.label :teamsport %>
-    <%= form.check_box :teamsport, id: :event_teamsport %>
-  </div>
-
-  <div class="field">
-    <%= form.label :playercount %>
-    <%= form.number_field :playercount, id: :event_playercount %>
-  </div>
-
-  <div class="field">
-    <%= form.label :gamesystem %>
-    <%= form.text_area :gamesystem, id: :event_gamesystem %>
```

Comment:
Also, here :) Where are those input-areas now?

Reply:
Isn't game_mode and gamesystem the same thing?